packages/schemas/src/seeds/setting.ts

```diff
@@ -9,7 +7,7 @@ export const createDefaultSetting = (): Readonly<CreateSetting> =>
   Object.freeze({
     id: defaultSettingId,
     adminConsole: {
-      language: Language.English,
+      language: 'en',
```

Comment:
should also fix the use of `Language` in packages/schemas/src/seeds/sign-in-experience.ts?

Reply:
hence we can remove the dependencies on @logto/phrases(-ui)?

Reply:
It is already done?